go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/ElrondNetwork/elrond-go
 
 go 1.13
 
+replace github.com/herumi/bls-go-binary => github.com/AdoAdoAdo/bls-go-binary v0.0.0-20200313082405-cbbb22d84df3
```

Comment:
What is this?

Reply:
To use a fork for now.

Reply:
this is until pull request is accepted in herumi/bls-go-binary

Reply:
But why the fork was not done on ElrondNetwork?

Reply:
it is only temporary.
It had the changes I did originally for upstream, AdoAdoAdo is my account.

Reply:
changed to github.com/ElrondNetwork/bls-go-binary v0.0.0-20200313082405-cbbb22d84df3